.studiorc

```diff
@@ -18,15 +18,11 @@ function install_from_source() {
 }
 
 function lint() {
-  pushd /src > /dev/null
-    shellcheck -s bash bin/* lib/*
-  popd > /dev/null
+  shellcheck -e SC1117 -s bash /src/bin/* /src/dot-studio/*
 }
 
 function syntax() {
-  pushd /src > /dev/null
-    bash -n bin/* lib/*
-  popd > /dev/null
+  bash -n /src/bin/* src/dot-studio/*
```

Comment:
Do we want these checks to recurse, or is the assumption that there will only be script files in these two directories, and none below?

Also, `bash -n` will only check files that have `#!/bin/bash` lines.

Reply:
We should only ever have a single depth `#!/bin/bash` file in those directories